### PR TITLE
Playground now deployments 🚀

### PR DESCRIPTION
**Pull request by emmenko**
Now it's easier to deploy the app on `now`, also for both production regions.

### CODE REVIEW

File: playground/now.eu.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "version": 1,
+  "public": true,
+  "name": "mc-extensions-channels-prod-eu",
+  "alias": "mc-extensions-channels-prod-eu",
+  "regions": ["bru"],
```
[the remark this answers is not in the document]
Comment (montezume) [reply playground/now.eu.json]: brussels I imagine? https://zeit.co/blog/multi-dc

**Comment by dferber90** [comment]
@emmenko Can you share the goal for deploying the playground? Who is the deployment for and why is the region difference important?

**Comment by emmenko** [comment]
@dferber90 it's mostly for testing and demo purposes.

The regions are for using it in MC EU/US.

**Comment by emmenko** [comment]
If you go to our "test with big data" project on prod you'll see the custom app 😉

For me it's very useful to test the extensibility infra part 😇

**Comment by tdeekens** [comment]
> For me it's very useful to test the extensibility infra part 😇

Ah super cool. Just didn't know about that part. 🚢

**Pull request merged**